src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+function operate(numberone, numbertwo, operation) {
+  const nmbrOne = new Big(numberone);
+  const nmbrTwo = new Big(numbertwo);
+  let result = 0;
+
+  switch (operation) {
+    case '+':
+      result = nmbrOne + nmbrTwo;
+      break;
+    case '-':
+      result = nmbrOne - nmbrTwo;
+      break;
+    case 'X':
+      result = nmbrOne * nmbrTwo;
+      break;
+    case '%':
+      result = nmbrOne / 100;
+      break;
+    case '&#247;':
+      result = nmbrOne / nmbrTwo;
```

Comment:
* [ ] Let us handle the division by zero, in this case
result = nmbrTwo === 0 ? 'undefined' : nmbrOne / nmbrTwo;